src/LlbcOfJson.ml

```diff
@@ -685,15 +698,18 @@ let fun_declaration_group_of_json (js : json) :
   combine_error_msgs js "fun_declaration_group_of_json"
     (g_declaration_group_of_json A.FunDeclId.id_of_json js)
 
-let global_declaration_group_of_json (js : json) (gid_conv : A.global_id_converter) :
+(* TODO Should a global declaration group be converted to its function bodies ?
+        It does not seems very clean.
+*)
```

Comment:
No, the global declaration group should be the globals themselves. The global bodies should really be considered as auxiliary definitions.